Dockerfile

```diff
@@ -20,11 +20,17 @@
 # docker run --net=host <image> -host-label=<label>
 ##
 
-FROM golang:1.4
+FROM golang:1.5.1
 MAINTAINER Madhav Puri <mapuri@cisco.com> (@mapuri)
 
+
+ENV http_proxy "http://proxy.esl.cisco.com:8080"
```

Comment:
I had to make Proxy additions for the following reason. I will remove cisco proxy values and have some other values.

make start
make[1]: Entering directory '/go/src/github.com/contiv/netplugin'
make[1]: Nothing to be done for 'start'.
make[1]: Leaving directory '/go/src/github.com/contiv/netplugin'
make ssh-build
make[1]: Entering directory '/go/src/github.com/contiv/netplugin'
cd /go/src/github.com/contiv/netplugin && make run-build
make[2]: Entering directory '/go/src/github.com/contiv/netplugin'
./scripts/deps
# cd .; git clone https://github.com/tools/godep /go/src/github.com/tools/godep
Cloning into '/go/src/github.com/tools/godep'...
fatal: unable to access 'https://github.com/tools/godep/': Failed to connect to github.com port 443: Connection timed out
package github.com/tools/godep: exit status 128